classes/migration/install/OMPMigration.inc.php

```diff
@@ -241,6 +268,11 @@ public function up(): void
             $table->unique(['path'], 'press_path');
         });
 
+        // DOI foreign key references Press, so needs to be added after
```

Comment:
Might be worth adding a comment where the doi tables are created, so someone reading the schema there will not that a foreign key is created later.